src/plus/launchpad/launchpadProvider.ts

```diff
@@ -609,15 +644,52 @@ export class LaunchpadProvider implements Disposable {
 		return repoRemotes;
 	}
 
+	mergeSearchedCategorizedItems(
+		permanent: LaunchpadCategorizedResult,
```

Comment:
What do we mean by `permanent` here?

Reply:
fixed: https://github.com/gitkraken/vscode-gitlens/pull/3684/commits/d44f3443cb11d938c01a71a65e24c20be09db999